CompuCell3D/core/pyinterface/CompuCellPython/CompuCell.i

```diff
@@ -293,6 +296,13 @@ using namespace CompuCell3D;
     s<<(*self);
     return s.str();
   }
+
+%pythoncode %{

```

Comment:
I think we could move this to SWIG interface - in CompuCell.i. . This  way we will keep C++ "pure" and do all pythonnic extensions like that in SWIG. I think Current swig file has examples of how to to it. I think CellG class is using %extend directive

Reply:
Maybe I'm not understanding this feedback, but is this requesting what's already implemented here? This addition is inside an %extend{} for Dim3D, and in CompuCell.i. Am I misunderstanding something?

Reply:
Right, I got fooled by how github presents changes. For some reason I thought this code was part of the *.cpp file